play-testkit/src/test/scala/akka/grpc/play/api/specs2/PlaySpecs2Spec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.play.api.specs2
+
+import org.junit.runner.RunWith
+import org.specs2.runner.JUnitRunner
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.{WSClient, WSRequest}
+import play.api.routing.Router
+import play.api.test._
+
+/**
+ * Test for the Play gRPC Specs2 APIs
+ */
+@RunWith(classOf[JUnitRunner])
+class PlaySpecs2Spec extends NewForServer with ServerGrpcClient with PlaySpecification with ApplicationFactories {
+
+  protected def applicationFactory: ApplicationFactory =
+    appFromGuice(GuiceApplicationBuilder().overrides(bind[Router].to[GreeterServiceImpl]))
```

Comment:
On a local fresh clone I'm getting compilation errors on missing `GreeterServiceImpl`. Is it possible we're missing a `src/test/proto/greetings.proto` or some tuning on dependencies?